static/scss/course-cards.scss

```diff
@@ -0,0 +1,105 @@
+// sass-lint:disable mixins-before-declarations
+.course-section h1 {
+  font-size: 44px;
+  font-weight: bold;
+  letter-spacing: 0;
+  line-height: 54px;
+  text-align: center;
+  padding-top: 54px;
+  padding-bottom: 54px;
+  margin: 0 auto;
+  max-width: 760px;
+
+  @include media-breakpoint-down(sm) {
+    font-size: 30px;
+    line-height: 40px;
+    padding-top: 20px;
+    padding-bottom: 22px;
+  }
+}
+
+.course-listing {
+  padding: 0;
+  margin: 0 -22px;
+  list-style: none;
+  letter-spacing: -0.32em;
+
+  .course-card-description {
+    overflow-y: scroll;
+    margin-bottom: 30px;
+  }
+
+  li {
+    @include media-breakpoint-down(md) {
+      width: calc(50% - 44px);
+    }
+
+    @include media-breakpoint-down(sm) {
+      width: calc(100% - 44px);
+    }
+    letter-spacing: normal;
+    display: inline-block;
+    vertical-align: top;
+    width: calc(33.33% - 44px);
+    margin: 0 22px 44px;
+
+    a {
+      background: white;
+      border-bottom: 3px solid transparent;
+      box-shadow: 0 1px 0 0 rgba(0, 0, 0, 0.23);
+      display: block;
+      text-decoration: none;
+
+      .course-info {
+        padding: 15px;
+
+        h2 {
+          height: 44px;
+          color: #000000;
+          font-size: 18px;
+          font-weight: 600;
+          letter-spacing: 0;
+          line-height: 22px;
+        }
+
+        p {
+          height: 115px;
+          color: #333333;
+          font-size: 16px;
+          font-weight: 400;
+          letter-spacing: 0;
+          line-height: 22px;
+        }
+
+        .date {
+          height: 19px;
+          color: #3e3e3e;
+          font-size: 16px;
+          font-weight: 400;
+          letter-spacing: 0;
+          line-height: 19px;
+        }
+      }
+
+      &:hover {
+        border-color: #a41e34;
+        box-shadow: 0 2px 15px 4px rgba(0, 0, 0, 0.16);
+
+        h2 {
+          color: #a41e34;
+        }
+      }
+    }
+
+    img {
+      width: 100%;
+      height: 223px;
+      display: block;
+      object-fit: cover;
+    }
+  }
+}
+
+.course-section {
+  background: #f8f8f8;
+}
```

Comment:
```suggestion
```

This isn't needed. The background is that color in the course section of the home page in the designs because there is a whole-page gradient that fades down to that color.

I just tested this locally and it seems to work. Change the `background` style of `body` in `layout.scss` to this, and we should get roughly the correct gradient, and it should fade to the correct color by the time the course section is rendered: 

`background: linear-gradient(to bottom, #ffffff, #f5f5f5 200px);`